crates/bevy_dev_tools/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#![warn(missing_docs)]
+//! This crate provides additional utilities for the [Bevy game engine](https://bevyengine.org),
+//! aimed at improving developer experience.
+
+use bevy_app::prelude::*;
+
+/// Adds developer tools to an App.
```

Comment:
We probably also want to add a note about how you should turn these off when shipping.